client/src/pages/spools/create.tsx

```diff
@@ -49,10 +50,17 @@ export const SpoolCreate: React.FC<IResourceComponentsProps & CreateOrCloneProps
     return {
       label: label,
       value: item.id,
+	    weight: item.weight,
+	    spool_weight: item.spool_weight
     };
```

Comment:
Please look into running Prettier to format the code before you commit it. It will automatically indent the lines appropriately etc so everything looks neat. If you use VS Code then that is very simple to setup.

Reply:
i dont use vs code. my ide defaults to tab = 4 spaces, ill change that and reformat the code i have added

Reply:
You can install prettier manually also and run it from the command line if you dont use vs code